src/domains/canoneunico/03_storage_account.tf

```diff
@@ -0,0 +1,65 @@
+module "cu_sa" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v6.15.2"
+
+  name                            = replace(format("%s-canoneunico-sa", local.project), "-", "")
+  account_kind                    = "StorageV2"
+  account_tier                    = "Standard"
+  account_replication_type        = "LRS"
```

Comment:
This must be a variable, because in prod we need to change into GRS

Reply:
done